asterisk_logcall/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Asterisk Phone Log-call
+=======================
+
+This bridge adds an option to the Asterisk IPBX servers for the URL to pull
+down call recordings. It also adds an option for time jitter compensation
+for use when automatically merging call records.
+
+Installation
+============
+
+To install this module, you need to simply click install.
+
+Configuration
+=============
+
+To configure this module, you need to go to the configuration page for the
+Asterisk IPBX server in question, within Odoo, make sure to edit the ucp_url
+field to be a url template for call recordings.
+
+The following macros are allowed:
+{odoo_type} (inbound, outbound)
+{odoo_src} (source phone number)
+{odoo_dst} (destination number)
+{odoo_duration} (length of call)
+{odoo_start} (start time of call in seconds since epoch)
+{odoo_filename} (file name on server)
+{odoo_uniqueid} (Asterisk UUID of call)
+
+Make sure to set time jitter compensation which is the number of seconds to
+subtract from new call start and add to new call end, for call merging, to
+compensate for system/database load and time drift between Asterisk server
+and Odoo/Odoo database server(s). 5 seconds is likely a good start. Above 10
+seconds you get into the realm where you may have distinct calls confused.
+20 - 30 seconds begins to guarantee this. It is best to keep this low and use
+a method to keep time synced.
+
+Known issues / Roadmap
+======================
+
+None at this time.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/connector-telephony/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Trever L. Adams
+
+Maintainer
+----------
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
```

Comment:
On 08/26/2016 01:22 AM, Eric Caudal wrote:
>
> In asterisk_logcall/README.rst
> <https://github.com/OCA/connector-telephony/pull/101#discussion_r76374465>:
>
> > +
> > +* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
> > +
> > +Contributors
> > +------------
> > +
> > +* Trever L. Adams
> > +
> > +Maintainer
> > +----------
> > +
> > +.. image:: https://odoo-community.org/logo.png
> > +   :alt: Odoo Community Association
> > +   :target: https://odoo-community.org
> > +
> > +This module is maintained by the OCA.
>
> I think I miss part of the standard footer. Can you check it again?
>
I am not sure how that happened. I messed up a copy and paste somehow. I
believe I have found and fixed all the places that happened.

Trever